src/scores/continuous/standard_impl.py

```diff
@@ -354,11 +354,10 @@ def pbias(
     """
     Calculates the percent bias, which is the ratio of the additive bias to the mean observed value, multiplied by 100.
 
-    Percent bias is used for evaluating and comparing forecast accuracy across stations or dataset with varying magnitudes.
-    By expressing the error as a percentage of the observed value, it allows for standardized comparisons, enabling assessment
-    of forecast performance regardless of the absolute scale of values. Like multiplicative_bias, Percent bias will return a np.inf
-    where the mean of `obs` across the dims to be reduced is 0.
-    It is defined as
+    Percent bias is used for evaluating and comparing forecast accuracy across stations or datasets with varying magnitudes.
+    By expressing the error as a percentage of the observed value, it allows for standardised comparisons, enabling assessment
+    of forecast performance regardless of the absolute scale of values. Like :py:func:`scores.continuous.multiplicative_bias`,
+    ``pbias`` will return a np.inf where the mean of ``obs`` across the dims to be reduced is 0. It is defined as
```

Comment:
```suggestion
    ``pbias`` will return a np.inf where the mean of ``obs`` across the dims to be reduced is 0. 
    
    It is defined as
```
You had "It is defined as" on a new line, but without a break. As such, in readthedocs, it was not rendering on a new line, it was rendering at the end of the preceding paragraph. (See https://scores.readthedocs.io/en/latest/api.html#scores.continuous.pbias).

Did you want "It is defined as" to be a new paragraph? If so, the suggestion above should do that. But if you wanted it to be part of the preceding paragraph, then this doesn't need further change.

Reply:
I think it's rendering fine how it is. If anyone wants to introduce a paragraph space, please feel free to raise this in a new PR, but it looks good to me.